src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -113,17 +115,23 @@ function ProviderSelectionField({
     () => {
       if (showProvidersList && (loadingProviders || loadingLocations)) {
         scrollAndFocus('.loading-indicator');
+      } else if (
+        showProvidersList &&
+        !loadingProviders &&
+        requestLocationStatus === FETCH_STATUS.failed
+      ) {
+        scrollAndFocus('#providerSelectionBlockedLocation');
       } else if (showProvidersList && !loadingProviders && !loadingLocations) {
         scrollAndFocus('#providerSelectionHeader');
       }
     },
     [loadingProviders, loadingLocations],
   );
 
-  return (
-    <div className="vads-u-background-color--gray-lightest vads-u-padding--2 medium-screen:vads-u-padding--3">
-      {!showProvidersList &&
-        !providerSelected && (
+  if (!showProvidersList) {
```

Comment:
I moved the non-list state up into its own return statement. I think in the future, we way to separate these into separate components.

Reply:
Yeah this file is certainly large enough